projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -133,9 +136,14 @@ public function initial_state() {
 		global $publicize;
 
 		return array(
-			'apiRoot'                          => esc_url_raw( rest_url() ),
-			'apiNonce'                         => wp_create_nonce( 'wp_rest' ),
-			'registrationNonce'                => wp_create_nonce( 'jetpack-registration-nonce' ),
+			'siteData'                         => array(
```

Comment:
I think this could break the ConnectScreen, as [it expects these values to be properties of the initial state object](https://github.com/Automattic/jetpack/pull/24203/files#diff-eb2d260a7e5ef7a49f634ad996a3c93135b2cc8cbe6f6209232bbf0670015dddL108).

Reply:
Yes, good catch! Will fix that.